src/router/routes/exportRoute.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+/* eslint-disable no-underscore-dangle */
+import express, { Router } from 'express';
+import { BulkDataAccess, ExportType, InitiateExportRequest } from 'fhir-works-on-aws-interface';
+import createHttpError from 'http-errors';
+import RouteHelper from './routeHelper';
+import ExportHandler from '../handlers/exportHandler';
+import ExportRouteHelper from './exportRouteHelper';
+
+export default class ExportRoute {
```

Comment:
I don't think this should be a class. 

It could just be a function like:
```
(serverUrl: string, bulkDataAccess: BulkDataAccess) => expressRouter
```
This class only has a public property that we care about (`router`) and a big constructor. It's not a good pattern.

I know that all other Routes follow this pattern but I don't think we should continue to do that.

Reply:
That makes sense. I think all the router class should follow the same pattern, whether it should be a function or a class. Therefore, I think if we change this class to a function, we should do with the other classes also. Can we leave this task as a future refactoring task?

Reply:
yes, it can be done on a separate PR